hack/bump-version.sh

```diff
@@ -22,12 +22,12 @@ if [[ "$OSTYPE" == "darwin"* ]]; then
 fi
 
 # parameters
-OPERATOR_OLD="v1\.2\.5"
-OPERATOR_NEW="v1\.2\.6"
-TIDB_OLD="v5\.2\.1"
-TIDB_NEW="v5\.3\.0"
-DM_OLD="v2.0.7"
-DM_NEW="v5.3.0"
+OPERATOR_OLD="v1\.2\.6"
+OPERATOR_NEW="v1\.2\.7"
+TIDB_OLD="v5\.3\.0"
+TIDB_NEW="v5\.4\.0"
+DM_OLD="v5.3.0"
+DM_NEW="v5.4.0"
```

Comment:
No need to update the version of TiDB and DM since TiDB Operator v1.2 only supports up to TiDB v5.3 and v5.3.0 is the latest version.